tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -893,12 +893,26 @@ native or user class, or enum. Nested array types (like ``Array[Array[int]]``) a
 Packed arrays
 ^^^^^^^^^^^^^
 
-GDScript arrays are allocated linearly in memory for speed.
-Large arrays (more than tens of thousands of elements) may however cause
-memory fragmentation. If this is a concern, special types of
-arrays are available. These only accept a single data type. They avoid memory
-fragmentation and use less memory, but are atomic and tend to run slower than generic
-arrays. They are therefore only recommended to use for large data sets:
+PackedArrays are generally faster to iterate on and modify compared to a typed
+Array of the same type (e.g. PackedInt32Array versus Array[int]) and consume
```

Comment:
```suggestion
Array of the same type (e.g. PackedInt64Array versus Array[int]) and consume
```
Isn't `int` is [64-bit](https://docs.godotengine.org/en/latest/classes/class_int.html#class-int) in GDScript?

Reply:
That part is taken verbatim from the class reference for the PackedArray classes (see: https://docs.godotengine.org/en/latest/classes/class_packedint64array.html which is where I got it from). I think it's just a general comparison. Default `int` might be 64-bit but we have PackedArrays for Int32 and Int64. Unless I've misunderstood what the correction is.

Reply:
Ah, it's probably fine then 👍

Reply:
I just saw your edits now. I think `Color` would be a less controversial class to use in the comparison.

PackedInt32Array/Array[int] isn't *wrong* it's just potentially confusing. Since this PackedInt32Array/Array[int] comparison already exists in the class reference, I'm personally okay with leaving it and merging this PR as-is. But I'll leave this conversation open so others can chime in if they want to.

Reply:
Github wasn't showing your comments after mine... I accepted the change. I'll just make the same adjustment to the class reference when I make a PR for that. After thinking about it, I felt that `Color` might be less "raw" of a comparison, since that's a more complex data type than an `int`. I felt keeping it to ints keeps it simpler, so changing it to PackedInt64Array feels cleaner to me.